sycl/test-e2e/Regression/same_unnamed_kernels.cpp

```diff
@@ -13,6 +12,7 @@
 //
 //===----------------------------------------------------------------------===//
 
+#include <sycl/accessor.hpp>
```

Comment:
Sorry I've missed this yesterday, but the specification says that we should only include `<sycl/sycl.hpp>` directly, everything else is implementation details.

```suggestion
```

Reply:
thanks, I seemed to require it at the time I added it, but it seems okay now.